packages/p2p/src/components/order-details/order-details.scss

```diff
@@ -144,19 +144,25 @@ $card-width: 456px;
         }
     }
     &__footer {
+        display: flex;
+        justify-content: space-between;
         border-top: 1px solid var(--general-section-1);
         margin-top: auto;
         font-size: 1.4rem;
         line-height: 1.43;
         color: var(--text-general);
         padding: 1.6rem 2.4rem;
-        display: flex;
         height: 72px;
 
         @include mobile {
-            border-top: unset;
-            height: 100%;
-            padding: unset;
+            width: 100%;
+            height: 72px;
```

Comment:
`height` has already been set to `72px` on line 155, why are we defining it again?

Reply:
> `height` has already been set to `72px` on line 155, why are we defining it again?

removed it and replaced px on rem.